### ISSUE TITLE
Execution failed for task ':proccessDebugManifest'.

### ISSUE THREAD

**Issue by webtestdeveloper** [pwlin/cordova-plugin-file-opener2#165]
![fileopener plugin issue](https://user-images.githubusercontent.com/22883784/30203956-798ab1b2-94a1-11e7-9500-fd67bf01cc6b.PNG)

**Comment by alonsoavotz** [comment]
same error here

**Comment by wkande** [comment]
I get the same error. I see some comment here on this issue but it does not seem to help.

https://github.com/pwlin/cordova-plugin-file-opener2/issues/152
`You must modify <framework src="com.android.support:support-v4:+" /> in plugin.xml to reflect your Android support version.`


Maybe I am not doing the suggested fix right or don't get it.

**Comment by moh3n9595** [comment]
same problem

**Comment by wkande** [comment]
Not knowing how to fix the supported version in the framework element as suggested I have come up with a temp fix that seems to work.

The following is from our internal project notes showing what worked for us. No guarantees, good luck.

------- INTERNAL NOTE --------

This plugin will fail to build with android@6.2.3. The build error cannot merge two android support
versions. It appears that one is declared by the app and one by the plugin, just a guess. Look at the plugin's `plugin.xml` file. There is an element for `framework`. The framework element declares a support version of -v4. This seems to suggest to the build to use android 26-alpha?

```
<framework src="com.android.support:support-v4:+" />
```

On the plugin's repo the developer comments in issue #152 as follows.

You must modify <framework src="com.android.support:support-v4:+" /> in plugin.xml to reflect your Android support version.

On the plugin's repo wkande comments in issue #165 that this did not seem to fix the issue probably because wkande does not know how to reflect the proper version.

The following fix (temp) seems to resolve the issue. The plugin's repo issue #165 has been updated and hopefully the developer will pick up on this and suggest a permanent repair or update the README to reflect the needed instructions for the build to succeed.

1. Remove the `framework` element from the plugin's plugin.xml file.
2. Remove the android platform from the project.
```
ionic cordova platform rm android
```
3. Add the android platform back into the project using the desired version.
```
ionic cordova plaform add android@6.2.3
```

This seems to fix the build issue and the app does work.

**Comment by sahyun1** [comment]
@wkande 
so I've removed the framework element from the plugin.xml.
Then remove and add back android version of 6.2.3 as you describe.
But still have the same issue.. did you use any other version of android?

**Comment by wkande** [comment]
No I did not use a different version, used:

ionic cordova plaform add android@6.2.3

It has been a while and it is possible something has changed with the plugin since then.

**Comment by shnist** [comment]
hi @webtestdeveloper, thanks for raising this issue. This issue has been resolved in the new release of 2.1.0, where you can optionally supply an android support version at runtime:
```
cordova plugin add cordova-plugin-file-opener2  --variable ANDROID_SUPPORT_VERSION={required version}
```
This should stop these compilation errors from happening.
